examples/config-overrides.js

```diff
@@ -1,6 +1,5 @@
 const { override } = require('customize-cra')
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
 
-module.exports = override(
-  //addWebpackPlugin(new BundleAnalyzerPlugin())
-)
+module.exports = override()
+//addWebpackPlugin(new BundleAnalyzerPlugin())
```

Comment:
let's revert this one as it looks weird.
```suggestion
module.exports = override(
  //addWebpackPlugin(new BundleAnalyzerPlugin())
)
```

Reply:
btw, I'm fine to remove the plugin entirely. furthermore, remove customize-cra entirely.

Reply:
Done.

I'm currently working on adding `@react-three/babel` which is still using `customize-cra`, this file on my branch is now like

```js
// examples/config-overrides.js
const { addWebpackPlugin, override, addBabelPlugin } = require('customize-cra')
const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin

const overrides = [
  addBabelPlugin('module:@react-three/babel'),
  process.env.BUNDLE_ANALYZE === 'true' ? addWebpackPlugin(new BundleAnalyzerPlugin()) : undefined,
]

module.exports = override(...overrides.filter((config) => config))
```

remove `customize-cra` then I'll need to add it back or have another approach, I'm leaving it there if no other reason to remove it.

Reply:
if you need it, no problem.